README.md

```diff
@@ -1,7 +1,6 @@
 # Twig Integration for Mezzio
 
-[![Build Status](https://travis-ci.com/mezzio/mezzio-twigrenderer.svg?branch=master)](https://travis-ci.com/mezzio/mezzio-twigrenderer)
-[![Coverage Status](https://coveralls.io/repos/github/mezzio/mezzio-twigrenderer/badge.svg?branch=master)](https://coveralls.io/github/mezzio/mezzio-twigrenderer?branch=master)
+[![Build Status](https://github.com/mezzio/mezzio-twigrenderer/workflows/continuous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml)
```

Comment:
Unfortunately, the image path is wrong:
```suggestion
[![Build Status](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml)
```

Compare:

![Build Status](https://github.com/mezzio/mezzio-twigrenderer/workflows/continuous-integration.yml/badge.svg)

![Build Status](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml/badge.svg)

Reply:
crap I copied my comment from the gist ... I'll have to correct this on quite a few, thanks

Reply:
corrected on all packages